src/vs/workbench/parts/quickopen/browser/gotoLineHandler.ts

```diff
@@ -213,7 +215,24 @@ export class GotoLineHandler extends QuickOpenHandler {
 	}
 
 	getAriaLabel(): string {
-		return nls.localize('gotoLineHandlerAriaLabel', "Type a line number to navigate to.");
+
+		let label = nls.localize('canNotRunPlaceholder', "This quick open handler can not be used in the current context.");
+
+		// Check we can run - (is text editor)
+		const canRun = this.canRun();
+		if ((typeof canRun === 'boolean' && !canRun) || typeof canRun === 'string') {
+			label = (typeof canRun === 'string') ? canRun : label;
+			return label;
+		}
+
+		const line = this.editorService.activeTextEditorWidget.getPosition().lineNumber;
+
+		if (typeof line === 'number') {
+			// Output verbose current line / go to line label
+			label = nls.localize('gotoLineLabelEmpty', "Current Line: {0}. Type a line number to navigate to", line) + '.';
```

Comment:
@cleidigh you cannot modify a string outside of the nls method, otherwise this would not be translated

Reply:
👍